examples/input_scene_interaction.py

```diff
@@ -0,0 +1,82 @@
+"""
+This example demonstrates slightly more complicated mouse and keyboard input
+in the interactive runtime viewport.
+"""
+
+from neoscore.core import neoscore
+from neoscore.core.brush import Brush
+from neoscore.core.key_event import KeyEventType
+from neoscore.core.mouse_event import MouseEventType
+from neoscore.core.path import Path
+from neoscore.core.pen import Pen
+from neoscore.core.text import Text
+from neoscore.core.units import Mm, Unit
+
+"""
+This example shows what keys are being held down and displays them on the center of the screen.
+It also allows the user to click-and-drag to generate line-segments.
+
+There are a few caveats that must be worked around to get this example working, both related
+to the viewport. The viewport by default shifts to accomodate new elements as they are added to the page.
+To restrict viewport movement, we create an arbitrarily large invisible rectangle.
+To generate line-segments where we click on the screen, the page coordinates and screen coordinates should be identical.
+One way to achieve this is by fixing the size and scale of the viewport.
+"""
+
+
+def key_handler(event):
+    """Add single characters to a list when a key is pressed,
+    and remove them from the list when the key is released.
+    """
```

Comment:
For this docstring and the other below, please update to consist of a short summary line, then longer explanation below as needed. Check out the docstrings elsewhere in the repo.